### ISSUE TITLE
Curses Should Auto Fire Back (Should be a Cost to Cursing)

### ISSUE THREAD

**Issue by Forman9** [jasonrohrer/OneLifeData7#1231]
'A curse is a mechanism by which you can say, "I don't want to play with this person anymore."'- Jason

If this is in fact the purpose of the system, then when you curse someone, they should automatically curse you back, without even the cost of a token. 

1. I think it is prema faci that everyone who has paid for the product has the right to enjoy the game in full - meaning not in donkeytown. 

2. If you REALLY don't want to play with someone, then YOU should have to accept the consequences of that decision, ie this decision should NOT override a paying customer's right to play.

3. People curse for silly reasons because there is NO COST to do so. If cursing a player results in an automatic curse back, people will be less likely to use and or abuse the system.

4. Sneak cursing a player (off screen) is tantamount to abusing the system. You are adding a token with potentially NO REASON, and at NO COST while preventing the other player from even knowing this happened or being able to curse back.

**Comment by Tarr-OHOL** [comment]
<img width="570" alt="Forman" src="https://github.com/jasonrohrer/OneLifeData7/assets/53706597/6fe59bed-f30c-4482-adf9-a4f86f940e62">

Is this you?

Curses don't block because the goal is to stop someone you find to be an issue from being around you, not to stop yourself from being around the person. The only people who find issue with the system are the trolls (you) 

1). You can still play the game if you're being an asshat. You just play alone for you know, being an asshat to others. People aren't playing games to enable people with shitty behavior to bully them lol. 

2). If you are being the problem what sense does it make to remove the non-issue player? For example: you flooded a town with nearly unkillable cows what sense does it make to bar the people cleaning up after you from the town? 

3). You have a 20 minute cooldown, if you curse willy nilly when you need the curse you can't use it. You're allowed to use your tokens just like anyone else. 

4). How is cursing off screen an issue? I do agree if you're blocked from a town the game should tell you this but you can figure this out by using /die.

**Comment by 4ferrocyanide** [comment]
I agree, curses need to be two way. If you don't want to play with somebody than you shouldn't have the option to be born with them vise versa. It's only fair under the current logic and it prevents curses from being abused by normal players and griefers.

**Comment by Forman9** [comment]
---or you being born near them." 

Totally forgot that he actually said that. 

So yeah, effectively the current system is just not at all doing what it is intended to do, in Jason's own words.

**Comment by Rukhmar** [comment]
No, otherwise people wouldn't curse and griefers might run rampant. The other answer is having people just kill the griefer repeatedly when they pop up which causes drama and newbies cursing anyone who kills anyone (even if it is completely justified). The curse is to punish the person that did the bad thing from coming back into that town to do bad thing again, not to punish the people reporting said person who did bad thing too.

**Comment by Rukhmar** [comment]
Jason wanted us to moderate ourselves basically so he can be more hands-off on the issue. Sometimes he needs to take a long break and doesn't want to look into every case of bad behavior. Also there is the issue of taking away a game someone paid for, so he made Donkey Town to let the bad actors still play with the mechanics of the game if they want, even if society has determined it is better for their survival to keep them away.

**Comment by 4ferrocyanide** [comment]
I wouldn't call this a fun alternative, a wasteland of dead cycle bots and whatever poor newbie was unlucky enough to get in here. Might as well be vestigial
![image](https://github.com/jasonrohrer/OneLifeData7/assets/167921232/79ef0c0f-76fd-4336-ad78-7d4d02efcf09)
![image](https://github.com/jasonrohrer/OneLifeData7/assets/167921232/312c9a14-c943-48e8-80a4-8b8e452824f7)

**Comment by Forman9** [comment]
> The curse is to punish the person that did the bad thing
But that's not what he said, he literally said it's a way to say you don't want to play with another player, and you likewise shouldn't be able to play near them.

> the person that did the bad thing
And who gets to make that determination? The use of a curse literally in NO WAY predicts whether or not it was used on someone who actually did a bad thing. More often I've seen them just get flung around over little dramas that pop off over berry munching or wall building.

**Comment by Forman9** [comment]
Just going to add more Jason quotes here from the trust update

"The curse system is great, insofar as it allows you to say, "I don't want to play near this person anymore," and have your wishes respected.  Each person gets to decide what counts as "unacceptable" on their own, without needed consensus from others.  Maybe one person doesn't like swearing, and another person doesn't like berry munching, and a third person really hates bossy players.  Some people might like the drama of a murder or theft, while others might hate it.  Curses allow us to avoid defining global behavior rules that must satisfy everyone."

"You will then see "+" symbols around their speech, and this effect persists across lives, with no time limit.  This doesn't allow you to track specific players across lives, but rather shows you the growing group of players that you generally trust."

**Comment by NotSoShady80** [comment]
I think that the long curse update was more of an I don't like you mechanic than a I want to banish you from the game mechanic because that would make all these different mechanics about war etc. vestigial and pretty worthless and in my opinion ruin the game's possibilities. That's why I agree with Forman.

**Comment by NotSoShady80** [comment]
I think that the long curse update was more of an I don't like you mechanic than a I want to banish you from the game mechanic because that would make all these different mechanics about war etc vestigial and pretty worthless and in my opinion it would ruin the game's possibilities.

That's why I agree with Forman. This update after all was meant to make curses more serious but with the two-way mechanic not working it made it easier for people to just do exactly what they were doing before the five years but, in a means to ban the user from the full OHOL experience and limit the game to what they think is the ideal way to play.

**Comment by GnihtonSeodYdahs** [comment]
Yeah but like I was saying people would not be throwing around curses because if they were two way they'd be alot more serious in them meaning that you hate a person.
